src/server/services/procedures/human-mortality-database/human-mortality-database.js

```diff
@@ -173,23 +173,20 @@ async function getData() {
 const mortality = {};
 
 /**
- * Get all the mortality data - potentially a lot of data.
- * Only use this if you truly need access to all data.
- * This is an object organized by country, then by date (mm/dd/yyyy), then by gender, then by category.
- *
- * Note: for countries that don't report separate male and female death counts,
- * the gender breakdowns are just the total multiplied by a rough estimate
- * of the percent of people in that country who are male/female.
+ * Get all the mortality data. This is potentially a lot of data.
+ * **Only use this if you truly need access to all data.**
+ * 
+ * This is returned as structured data organized by country, then by date (mm/dd/yyyy), then by gender, then by category.
  *
- * @returns {Array}
+ * @returns {Array} all available data
  */
-mortality.getAllData = getData;
+mortality.getAllData = async () => await getData();
```

Comment:
Why was this change necessary? It isn't a problem but I am curious...

Reply:
Oh, that was an issue where apparently the jsdoc extractor wasn't matching that syntax. I fixed it before the other jsdoc syntax thing and forgot about it. I can go back and just add another regex for alias rpcs like this.